src/Regulator.cpp

```diff
@@ -134,16 +285,8 @@ Regulator::Regulator(int rcvChannels, int bit_res, int FPP, int qLen, int bqLen,
     , m_b_BroadcastRingBuffer(NULL)
     , m_b_BroadcastQueueLength(bqLen)
 {
-    // catch settings that are compute bound using long HIST
-    // hub client rcvChannels is set from client's settings parameters
-    // hub server rcvChannels is set from connecting client, not from hub parameters
-    //    if (mNumChannels > MaxChans) {
-    //        std::cerr << "*** Regulator.cpp: receive channels = " << mNumChannels
-    //                  << " larger than max channels = " << MaxChans << "\n";
-    //        exit(1);
-    //    }
-    if (mFPP > MaxFPP) {
-        std::cerr << "*** Regulator.cpp: local FPP = " << mFPP
+    if (mLocalFPP > MaxFPP) {
```

Comment:
seems like this check should maybe be for `mPeerFPP`, after it's set in `setFPPratio`?

Reply:
old check, I'll remove it -- unless we fear devices that are greater than 1024